happi/device.py

```diff
@@ -382,3 +356,44 @@ def __repr__(self):
 
     def __eq__(self, other):
         return (self.prefix, self.name) == (other.prefix, other.name)
```

Comment:
Not part of the PR, but this jumps out to me as being insufficient.
It should be something like:
`(type(self) is type(other)) and all((self.* == other.*))`

Reply:
Good point. Let's make an issue

Reply:
actually adressing this now, `current  __eq__ depends on all devices having a prefix